### PR TITLE
Fe/lab4

### PR DESCRIPTION
**Pull request by KirylAkulich1**

### CODE REVIEW

**Comment by KirylAkulich1** [comment]
https://kirylakulich1.github.io/FrontEndTasks/fe_lab3/
To navigate to admin panel press SHIFT+TAB anywhere and enter login:superadmin@super.com  password : superadmin

File: fe_lab3/css/styles.css [hunk 1]
```diff
@@ -150,3 +180,70 @@ section{
...
+  margin: 15% auto; /* 15% from the top and centered */
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%; /* Could be more or less, depending on screen size */
+}
+
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
```
Comment (gmltA) [inline fe_lab3/css/styles.css:223]: Not allowed.

**Review by gmltA — CHANGES_REQUESTED** [review]

File: fe_lab3/css/styles.css (hunk 1 above)
Comment (KirylAkulich1) [reply fe_lab3/css/styles.css:223]: is this the only place I need to fix to pass lab4?

Comment (KirylAkulich1) [reply fe_lab3/css/styles.css:223]: Is it the only place I need to fix to pass lab4?

Comment (gmltA) [reply fe_lab3/css/styles.css:223]: If there are other occurrences of `float` then you need to replace them too.

Comment (gmltA) [reply fe_lab3/css/styles.css:223]: But yes, that would be enough to pass.

Comment (KirylAkulich1) [reply fe_lab3/css/styles.css:226]: fixed

**Review by gmltA — APPROVED** [review]
That's a <kbd>7</kbd>.

**Comment by KirylAkulich1** [comment]
Thank you

**Pull request closed without merging**